### DISCUSSION TITLE
使用xhofe/alist-aria2镜像，aria2可以离线下载，但是下载完成后会出现transfer，然后下载的数据被清除了

### DISCUSSION THREAD

**Discussion by MileWay** [AlistGo/alist#4722] [Q&A]
download http://xxxx.xxxx.xxxx/d/aliyun/%E6%9C%89%E5%A3%B0%E4%B9%A6/%E5%BE%97%E5%88%B0VIP%E8%B5%A0%E4%B9%A6/%E5%BF%85%E7%84%B61/%E5%BF%85%E7%84%B61.jpg to [/aliyun](/有声书/得到VIP赠书/必然1)
成功
completed

将下载的文件传输到相应的存储
运行中...
已完成
transfer /opt/alist/data/temp/aria2/1657bc4e-9a38-4fb4-b23e-a4058fd56f25/必然1.jpg to [/aliyun](/有声书/得到VIP赠书/必然1)
成功

是哪里配置不对吗，应该要怎样配置呢？

**Comment by MileWay** [comment]
有人吗？帮忙看下啊

**Comment by sjdskl** [comment]
下载后又被传回阿里云了。然后选择下载的时候下面有个选项是上次后删除。默认上传后就删除了。所以不见了。我也被这个搞得摸不到头脑。
